components/centraldashboard/frontend/index/components.html

```diff
@@ -0,0 +1,22 @@
+<base href="https://cdn.rawgit.com/download/polymer-cdn/2.6.0/lib/">
```

Comment:
We should move away from using the CDN.
Separately, rawgit is going away, we shouldn't rely on it even in the interim.